docs/overview.rst

```diff
@@ -80,6 +80,14 @@ Run all or selected steps for each plan::
     tmt run discover
     tmt run prepare execute
 
+See any failures from a previous run::
+
+    tmt run --list report -vvv
```

Comment:
Seems like a typo, `--list` instead of `--last`.

Reply:
Silly me - Thank you! I fixed this and (force) pushed the tweak again.